CHANGELOG.rst

```diff
@@ -17,6 +17,9 @@ removed
 changes
 =======
 
+-  The ``wx_property_tag`` validator now also accepts lists of different tags. [:pull:`670`]
+   When multiple tags are passed, validation will pass if *one or more* of the supplied pattern matches.
```

Comment:
```suggestion
   When multiple tags are passed, validation will fail if *none* of the supplied patterns match.
```
The previous formulation sounded like everything passes as long as I have one of the required tags